rpm_spec/subpackages/manageiq-gemset

```diff
@@ -40,6 +40,7 @@ Requires: v2v-conversion-host-ansible
 
 # For Appliance Console
 Requires: network-scripts
+Requires: libsodium-devel
```

Comment:
Technically it's needed for the google provider too now, I think?  @agrare does this look correct to you?

Also, @jvlcek, can you alphabetize here?

Reply:
:+1: LGTM

Technically:
JWT optionally uses rbnacl if it is installed
googleauth uses JWT
google-api-ruby-client uses googleauth
fog-google uses google-api-ruby-client
manageiq-providers-google uses fog-google
we use manageiq-providers...just kidding :wink:

Reply:
LOL! You're evil @agrare! I like it!

Reply:
Yeah that was a "fun" one to track down